lib/email_tracking_retriever.py

```diff
@@ -170,6 +170,10 @@ def get_items_from_email(self, email_str) -> Any:
   def get_delivery_date_from_email(self, email_str) -> Any:
     pass
 
+  @abstractmethod
```

Comment:
we can remove the abstractmethod annotation here, then we don't need to implement or refer to this method at all in the best buy tracking retriever